resources/lang/en/thoughts/alive/deputy.json

```
@@ -269,7 +275,9 @@
     {
       "id": "healer_deputy_to_medcat",
         "thoughts": [
-            "Takes a patrol out to gather herbs for the medicine cat den"
+            "Takes a patrol out to gather herbs for the medicine cat den",
+            "Insists that r_c teaches the clan about basic medicinal treatments",
+            "Explains how to treat a simple battle injury with r_c watching, impressed"
```

```suggestion
            "Explains how to treat a simple battle injury while r_c watches, impressed"
```
this flows a bit nicer